config/condor_mapfile

```diff
@@ -1,6 +1,6 @@
 GSI "^\/DC\=com\/DC\=DigiCert-Grid\/O=Open Science Grid\/OU\=Services\/CN\=(host\/)?([A-Za-z0-9.\-]*)$" \2@daemon.opensciencegrid.org
 GSI "^\/DC\=DigiCert-Grid\/DC\=com\/O=Open Science Grid\/OU\=Services\/CN\=(host\/)?([A-Za-z0-9.\-]*)$" \2@daemon.opensciencegrid.org
 GSI (.*) GSS_ASSIST_GRIDMAP
-GSI (.*) anonymous@gsi
+GSI (/CN=[-.A-Za-z0-9/= ]+) \1@unmapped.opensciencegrid.org
```

Comment:
@bbockelm, I'm noticing errors in my SchedLog with the newest version of HTCondor CE:

> 03/17/15 16:29:10 ERROR: Error compiling expression '(/CN=[-.A-Za-z0-9/=' -- missing terminating ] for character class.  this entry will be ignored.

What is this regex supposed to be matching? should we replace the ' ' with '\s' to stop condor from complaining?